lib/ansible/modules/files/yedit.py

```diff
@@ -0,0 +1,965 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# pylint: disable=wrong-import-order,wrong-import-position,unused-import
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'core'}
```

Comment:
community, no core here please.

Reply:
I was copying what another file had in it.  I'll update.